ActionNetworkContactImporter.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+require_once("AbstractContactImporter.php");
+
+class ActionNetworkContactImporter extends AbstractContactImporter
+{
+
+    private $client;
+    private $explorer;
+    private $adapter;
+
+    // constructor
+    public function __construct($endpoint, $schema, $apikey) {
+        self.endpoint = $endpoint;
+        self.schema = $schema;
+        self.apikey = $apikey;
+
+        // demo api key is c173bee8d3238bb5bfc4dd014f207feb.
```

Comment:
I'd not commit this to source code -- probably should regenerate your key :)

Reply:
looool revoked it